### DISCUSSION TITLE
hi

### DISCUSSION THREAD

**Discussion by massimo1220** [community/community#33399] [Other Feature Feedback, Questions, & Ideas] [answered]
hi, How can I get "brain" in github?

**Comment by crypto-demigod** [comment, marked as answer]
@DevStar0202 , to achieve the 'Galaxy Brain' achievement, take note of the following:

Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.
Hope this answer has helped you, and all the best in getting the achievement! :)

**Reply by massimo1220** [reply]
Thanks for your kindly reply.

**Reply by crypto-demigod** [reply]
Anyhow, it's not difficult for you. see the others then answer. it's simple.

**Reply by massimo1220** [reply]
yeah

**Reply by crypto-demigod** [reply]
how can I get "merge" in github.com?

**Reply by massimo1220** [reply]
you can just request "pull request". or you can create new brand in your project then you must request pull request.
then push merge button.

**Reply by crypto-demigod** [reply]
I got the point. good answer.

**Reply by crypto-demigod** [reply]
thanks.

**Reply by massimo1220** [reply]
you are welcome.
